specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13936,6 +16885,10 @@
         }
       }
     },
+    "Object": {
```

Comment:
ContinuousWebJob has a field, "settings", for which we can take arbitrary JSON.  I'm not to sure of the details, but I believe this element is a result of how the tooling spews Swagger.